### DISCUSSION TITLE
Alta calidad

### DISCUSSION THREAD

**Discussion by JuanEduardoMHz** [community/community#136915] [Other Feature Feedback, Questions, & Ideas]
¿Cómo tener alta calidad audio y vídeo en un Motorola y una televisión inteligente uhd Samsung?

**Comment by JuanEduardoMHz** [comment]
4k hdr

**Comment by Yackens** [comment]
Para obtener una alta calidad de audio y vídeo cuando utilizas tu Motorola con una televisión inteligente UHD Samsung, sigue estos pasos:
- Conexión mediante HDMI
- Conexión Inalámbrica (Screen Mirroring o Chromecast)
- Ajustes de Resolución y Calidad de Audio (4H UHD y audio Dolby o DTS)
- Red Wi-Fi de Alta Velocidad (mejor usa conexion por cable para que al ver videos o pelis de internet no se vea con una resolución más baja)

¡Espero haberte ayudado! Saludos.

**Comment by samus-aran** [comment]
Hi @JuanEduardoMHz ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
